sdk/inc/azure/iot/az_iot_hub_client.h

```diff
@@ -233,6 +233,19 @@ typedef struct
   } _internal;
 } az_iot_hub_client_properties;
 
+/**
+ * @brief Supported IoT Hub properties
+ */
+static const az_span az_iot_hub_client_properties_message_id
+    = AZ_SPAN_LITERAL_FROM_STR("%24.mid"); /**< Add unique identification to a message */

```

Comment:
>%24.mid [](start = 32, length = 7)

Can these be macro ... if someone do not want span?

Reply:
yea, we should collect these in a header file

Reply:
This also gets to a larger point which I was going to make later with PnP that the properties stuff should be pulled out maybe into common. Since this stuff (properties append, create, etc) is the same whether someone is using PnP or the Hub Client. Do I move this all now?

Reply:
As for the macro yes we could. I took the best guess on the type but open to other's. In Azure RTOS' case it's probably easier as a macro to then use as a `char*`.

Reply:
we already have a iot common, use that